nixpkgs has a more recent rnnoise than the one used by mumble, and rnnoise
changed the argument rnnoise_create[1],

[1] https://github.com/xiph/rnnoise/commit/231b9c02d14a74cb449a98004cb7a2cf1bdeca2f

--- old/src/mumble/AudioInput.cpp	2020-02-18 22:55:32.000000000 -0500
+++ new/src/mumble/AudioInput.cpp	2020-02-18 22:58:08.000000000 -0500
@@ -106,7 +106,7 @@
 #endif
 
 #ifdef USE_RNNOISE
-	denoiseState = rnnoise_create();
+	denoiseState = rnnoise_create(NULL);
 #endif
 
 	qWarning("AudioInput: %d bits/s, %d hz, %d sample", iAudioQuality, iSampleRate, iFrameSize);
